### PR TITLE
✨Entrance

### LINKED ISSUE #40: Entrance Level

### PR DESCRIPTION
**Pull request by davey2206**
closes #40 
Added first version of the entrance level

image

### CODE REVIEW

File: Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat [hunk 1]
```diff
@@ -36,6 +36,7 @@ Material:
     RenderType: Transparent
   disabledShaderPasses:
   - DepthOnly
+  - SHADOWCASTER
```
Comment (NathanThus) [inline Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat:39]: Why is this disabled?

File: Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat [hunk 2]
```diff
@@ -133,7 +134,7 @@ Material:
     - _ZWrite: 0
     m_Colors:
     - _BaseColor: {r: 0.6985294, g: 0.82535493, b: 1, a: 0.24705882}
-    - _Color: {r: 1, g: 1, b: 1, a: 1}
+    - _Color: {r: 0.6985294, g: 0.82535493, b: 1, a: 0.24705882}
```
Comment (NathanThus) [inline Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat:137]: Why is this changed?

**Review by NathanThus — CHANGES_REQUESTED** [review]

File: Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat (hunk 2 above)
Comment (davey2206) [reply Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat:137]: Color of the water is still good

File: Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat (hunk 1 above)
Comment (NathanThus) [reply Assets/Art/PolygonAdventure/Materials/PolyAdventureMaterial_Water_01.mat:39]: This seems to be benign, so this is resolved.

**Review by NathanThus — APPROVED** [review]
